### PR TITLE
#268: add flag support for ocamlmklib options

### PR DESCRIPTION
**Pull request by gasche**
See #268

### CODE REVIEW

File: src/ocaml_specific.ml [hunk 1]
```diff
@@ -1020,4 +1025,15 @@ begin
...
 end;;
 
+begin
+  let ocamlmklib_pflag ?doc_param flag =
+    pflag ["ocamlmklib"] flag ?doc_param
+      (fun param -> S [A ("-"^flag); A param]) in
+  ocamlmklib_pflag "cclib";
+  ocamlmklib_pflag "ccopt";
+  ocamlmklib_pflag "dllpath";
+  ocamlmklib_pflag "rpath";
```
Comment (whitequark) [inline src/ocaml_specific.ml]: I don't think we need both `-dllpath` and `-rpath` since those are aliases. I would say keep only `-rpath`.

Comment (gasche) [reply src/ocaml_specific.ml]: Thanks for the comment. My idea was that windows users may find "dllpath" more natural, and unix users prefer "rpath". I'll move to rpath-only.

Comment (whitequark) [reply src/ocaml_specific.ml]: Windows doesn't have the concept of rpath.

**Comment by gasche** [comment]
Thanks!

**Pull request merged**